pyproject.toml

```diff
@@ -66,8 +65,7 @@ coverage = {extras = ["toml"], version = ">=6.1,<8.0"}
 optional = true
 
 [tool.poetry.group.pre-commit.dependencies]
-black = ">=22.3"
-pylint = ">=2.13,<4.0"
+mypy = ">= 0.800"
```

Comment:
I think mypy should not be in this list, as it is not run in the precommits?

Reply:
Ah, yep. It's already in the mypy group which is also installed on the CI action